main/resource-access.bs

```diff
@@ -75,6 +75,291 @@ Issue: Write Linked Data Platform section.
+header. Conversely, the resource being described by a Descriptive metadata
+resource is discovered by the client via the `rel="describes"` `Link` relation
+in a `Link` header.
+
+A given Solid resource MUST NOT be directly associated with more than one
+Descriptive metadata resource.
+
+To create or modify a Descriptive metadata resource, a given
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Write](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the [ACL inheritance
+algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+To discover or read a Descriptive metadata resource, an
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Read](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the [ACL inheritance
+algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+#### Shape Validation #### {#rm-shape}
+
+Issue: Detail the composition of a shape metadata resource, including the shape
+language, shape url, and any additional parameters to be used in shape
+validation by the server implementation.
+
+Shape Validation ensures that any data changes in a given resource conform to an
+associated [[!SHACL]] or [[!SHEX]] data shape. It MUST be supported as a
+resource metadata type by Solid servers.
+
+The Shape validation metadata resource directly associated with a given resource
+is discovered by the client via the `rel=http://www.w3.org/ns/solid/terms#shape`
+`Link` relation in a `Link` header. Conversely, the resource being described by
+a Shape validation metadata resource is discovered by the client via the
+`rel=describes` `Link` relation in a `Link` header.
+
+A given Solid resource MUST NOT be directly associated with more than one
+Descriptive metadata resource.
+
+To create or modify a Shape validation metadata resource, an
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Write](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the [ACL inheritance
+algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+To read a Shape validation metadata resource, an
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Read](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the [ACL inheritance
+algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+A Solid server SHOULD sanity check Shape validation metadata resources upon
+creation or update to restrict invalid changes.
```

Comment:
So shape validate the shape?

Reply:
[ShExR.shex](https://github.com/shexSpec/shexTest/blob/master/doc/ShExR.shex) is pretty fussy. [SHACL.shacl](https://www.w3.org/TR/shacl/#shacl-shacl) had issues around performance but I think they backed off on the expressivity enough to make it reasonable.

Reply:
See prior comment - the auxiliary resource is not the shape itself, it is a pointer to the shape that should be used for validation along with other parameters. There is another issue inline above this to provide the shape that this resource should conform to.